utils.sh

```diff
@@ -674,15 +674,4 @@ function auth_template_and_removetmp(){
     removetmp
 }
 
-use_podman_registry() {
```

Comment:
Maybe I'm missing something but isn't this function used twice in 02_configure_host.sh
https://github.com/openshift-metal3/dev-scripts/blob/master/02_configure_host.sh#L71 AND
https://github.com/openshift-metal3/dev-scripts/blob/master/02_configure_host.sh#L285

And its use was only removed at line 71 so the function can't be removed yet (or the other use needs to be changed).

Not sure if that is causing any problems although seeing a lot of strange test failures.